apps/nextjs-website/src/_contents/products.ts

```diff
@@ -5,11 +5,12 @@ import { pagoPaGuides } from '@/_contents/pagoPa/guides';
 import { appIO } from './appIo/appIO';
 import { ioSign } from './ioSign/ioSign';
 import { pagoPa } from './pagoPa/pagoPa';
+import { ioSignGuides } from '@/_contents/ioSign/guides';
 
 export const overviews = [appIoOverview, ioSignOverview, pagoPaOverview];
 export const quickStarts = [];
 export const apis = [];
 export const tutorials = [];
 export const tools = [];
-export const productsGuides = [pagoPaGuides];
+export const productsGuides = [pagoPaGuides, ioSignGuides];
```

Comment:
The `productsGuides` is going to be ambiguous once the gitbook guides are integrated. What do you think to rename it to `guideHomepages`?

Reply:
What about `guideLists`?

Reply:
Perfect! 🚀